.github/workflows/add-incident.yml

```diff
@@ -0,0 +1,26 @@
+name: Add comment
```

Comment:
Can we name this better? Add comment is a bit generic...

Reply:
How does `Add incident metrics` sound ?

Reply:
What about...

```suggestion
name: Add incident detail template
```